Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/11-blue-green.md

```diff
@@ -0,0 +1,271 @@
+}
+
+```
+
+After the deployment if the Landing Zone, install a sample application to test the deployment. The sample application to use is stored in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\07-AKS-cluster\sample-workload-for-agic-test.yaml".
+
+```bash
+az aks command invoke --resource-group $ClusterRGName --name $ClusterName   --command "kubectl apply -f sample-workload-for-agic-test.yaml "
+```
+
+As an alternative you can run the kubectl command directly from the Linux jump box access of the BastionHost provisoned as part in the [hub network](./04-network-hub.md).
+
+If you have deployed the public dns zone that is part of [AKS-Supporting](./06-aks-supporting.md), than you can test the deployment, executing an invocation to the sample app.
+
+```bash
+curl http://{hostname-app}.{public_domain}/
+```
+
+If the azure public dns zone is not attached to the domain, than is possible to test the app endpoint with the followwing command.
+
+```bash
+curl -H "Host: {hostname-app}.{public_domain}" http://{app-gateway-pip}/
+```
+
+## T1: Green Cluster Deployment
+
+At this stage is required to perfor the following action to deploy the new green cluster in co-existence with the blue one.
+
+1. Run again the flow mentioned [here](./05-network-lz.md), with the following configuration in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\05-Network-LZ\app-gateway.tf"
+
+```
+locals {
+  appgws = {
+    "appgw_blue" = {
+      name_prefix="blue"
+      appgw_turn_on=true
+    },
+    "appgw_green" = {
+      name_prefix="green"
+      appgw_turn_on=true
+    }
+  }
+}
+
+```
+
+2. Run the flow mentioned [here](./07-aks-cluster.md), with the following configuration in the file "Scenarios\AKS-Secure-Baseline-PrivateCluster\Terraform\07-AKS-cluster\aks-cluster.tf"
+
+```
+locals {
+  aks_clusters = {
+    "aks_blue" = {
+      name_prefix="blue"
+      aks_turn_on=true
+      k8s_version="1.23.5"
+      appgw_name="lzappgw-blue"
+    },
+    "aks_green" = {
+      name_prefix="green"
+      aks_turn_on=true
+      k8s_version="1.23.5"
+      appgw_name="lzappgw-green"
+    }
+  }
+}
+
+```
+
+## T2: Sync K8S State between Blue and Green clusters
+
+In our sample scenario, this means deploy the sample workload and related K8S resources in the green cluster.
+
+```bash
+az aks get-credentials --resource-group $ClusterRGName --name $GreenClusterName
+az aks command invoke --resource-group $ClusterRGName --name $GreenClusterName  --command "kubectl apply -f sample-workload-for-agic-test.yaml "
+```
+
+after the deployment you can test the application wit the following command.
+
+```bash
+curl -H "Host: {hostname-app-green}.{public_domain}" http://{app-gateway-pip}/
+```
+where app-gateway-pip is the public ip of the app gateway. 
+If the validation is ok, than the new cluster can be promoted as new production/stable cluster. Follow the instruction described in the [next section](#t3-traffic-switch-to-the-green-cluster).
```

Comment:
Change to "where app-gateway-pip is the public ip of the green app gateway"

Reply:
fixed